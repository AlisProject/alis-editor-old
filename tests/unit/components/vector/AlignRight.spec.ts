import { mount } from '@vue/test-utils'
import AlignRight from '@/components/vector/AlignRight.vue'

describe('AlignRight.vue', () => {
  test('renders correctly', () => {
    const wrapper = mount(AlignRight)
    expect(wrapper.element).toMatchSnapshot()
  })
})
