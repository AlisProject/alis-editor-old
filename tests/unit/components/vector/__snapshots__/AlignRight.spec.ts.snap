// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AlignRight.vue renders correctly 1`] = `
<svg
  class="feather feather-align-right"
  fill="none"
  height="24"
  stroke="currentColor"
  stroke-linecap="round"
  stroke-linejoin="round"
  stroke-width="2"
  viewBox="0 0 24 24"
  width="24"
  xmlns="http://www.w3.org/2000/svg"
>
  <line
    x1="21"
    x2="7"
    y1="10"
    y2="10"
  />
   
  <line
    x1="21"
    x2="3"
    y1="6"
    y2="6"
  />
   
  <line
    x1="21"
    x2="3"
    y1="14"
    y2="14"
  />
   
  <line
    x1="21"
    x2="7"
    y1="18"
    y2="18"
  />
</svg>
`;
